api/graphql/types/allocated_time_slot/permissions.py

```diff
@@ -57,5 +52,4 @@ def has_delete_permission(cls, instance: AllocatedTimeSlot, user: AnyUser, input
         if not user.has_staff_permissions:
             return False
 
-        sector = instance.reservation_unit_option.application_section.application.application_round.service_sector
-        return can_manage_service_sectors_applications(user, sector)
+        return can_manage_service_sectors_applications(user)
```

Comment:
Same comment as create, we can do away with `can_manage_service_sectors_applications` here too.

Reply:
Done.